style.css

```diff
@@ -0,0 +1,467 @@
+header {
+  width: 100%;
+  height: 60px;
+  padding: 0 16px;
+  background-color: rgb(245, 235, 235);
+  position: fixed;
+  top: 0;
+}
+
+nav {
+  display: flex;
+  height: 100%;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.start-pt {
+  display: flex;
+  height: 100%;
+  align-items: center;
+}
+
+.bars-info {
+  margin-right: 16px;
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+}
+
+.nav-brand {
+  width: 80px;
+}
+
+.center {
+  display: flex;
+  width: 40%;
+  height: 100%;
+  align-items: center;
+}
+
+.search-bar {
+  display: flex;
+  width: 100%;
+  align-items: center;
+  border: 1px solid;
+  border-color: lightgray;
+}
+
+.search-bar input[type=text] {
+  flex: 1;
+  padding: 4px 6px;
+  font-size: 16px;
+  font-family: "Roboto", "Noto", sans-serif;
+  font-weight: 400;
+  line-height: 24px;
+  border: none;
+  border-right: 1px solid;
+  border-color: lightgray;
+}
+
+.search-bar button {
+  padding: 4px 8px;
+  background: #f8f8f8;
+  border: none;
+  cursor: pointer;
+}
+
+.nav-right {
+  display: flex;
+  align-items: center;
+  margin-right: 32px;
+}
+
+.nav-right button {
+  margin-right: 8px;
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+}
+
+.nav-ico {
+  width: 32px;
+  height: 32px;
+}
```

Comment:
Please add the same Icon as Youtube and margins to your design and try to stick as much as possible to the given design
![youtube 1](https://user-images.githubusercontent.com/97598663/159269502-14b2af3e-72b4-4294-94b5-d287e227a846.jpg)